src/client/pages/ConnectPayment/components/TrustlyModal.tsx

```diff
@@ -9,11 +9,11 @@ import { useTextKeys } from 'utils/textKeys'
 const Header = styled('div')({
   width: '100%',
   height: 40,
-  background: colors.DARK_GREEN,
+  background: '#009175',
```

Comment:
Do we use this component anymore? Didn't want to add this color to the brand colors since it's more related to matching the Trustly colors I suppose

Reply:
we use this in the SE flow if i'm not mistaken. but we only use the color here.